pupa/scrape/schemas/vote.py

```diff
@@ -7,7 +7,7 @@
     "properties": {
         'identifier': {"type": "string", "blank": True,},
         'motion_text': {"type": "string" },
-        'motion_classification': {"items": {"type": "string", "enum": common.VOTE_CLASSIFICATIONS},
+        'motion_classification': {"items": {"type": "string"},
```

Comment:
Euch, I just saw your comment after I merged, sorry @jamesturk 

(but I think we agree, and I filed a bug #153 )